content/virtualization.md

```diff
@@ -0,0 +1,199 @@
+
+### Copy and Paste
+
+Once your VM has the `spice-vdagent` or an equivelent package installed on your VM, reboot the VM once to load this driver. At this point, you will now be able to copy and paste between your VM and the Host just like copy and pasting between applications.
+
+### Share a USB Device
+
+![Devices and Shares](/images/gnome-boxes/boxes-vmprefs-devices_and_shares.png)
+
+One of the features of GNOME Boxes, is the ability to transfer a USB device connected to your Host to the VM. You can find this option in the VM settings menu, under the "Devices & Shares" tab. On this tab, under the "USB devices" heading, you will see a list of the USB devices found on your system. If you click the toggle bottom on the right, the USB device will disappear from your Host operating system, and it will appear inside your VM.
+
+### Share a Folder
+
+_For this guide, we used GNOME-based Linux distributions, so the VMs UI closely matched Pop!_OS and Ubuntu. Accessing shared folders with other desktop environments is possible, but you will need to refer to their documentation for the details_
+
+![Folder Shares](/images/gnome-boxes/boxes-folder-shares.png)
+
+To share a folder from the Host with your VM, you will need the `spice-webdavd` or an equivelent package installed on your VM. From the VM settings menu, select the "Devices and Shares" tab and scroll to the bottom. At the bottom of this menu, you will see the "Folder Shares" section. Click the <kbd>+</kbd> button to show the folder share pop-up.
+
+#### Selecting the Host's Folder
+
+In this pop-up, the "Local Folder" refers to the folder's name on the Host system. This defaults to the "Public" folder in your Home directory. If you click the drop down, you can quickly select some of the important folders on your system. If you do not wish to use one of these folders, select "Other..." This will open a folder selection window, where you can then navigate to the folder you wish to share with the VM. Once you are in the folder you wish to share, click "Open" in the top right corner.
+
+#### Accessing a Shared Folder
+
+![Accessing a Shared Folder](/images/gnome-boxes/boxes-vm-access-folder-share.png)
+
+Now that GNOME Boxes is sharing the folder, we can close the VM settings menu and move back to the VM. In the VM, open your Files application, then navigate to "Other Locations" on the left hand side. You will likely see a spinning loading wheel under "Networks" for a moment, so wait for that to finish. Once the loading is done, you should see "Spice client folder" listed under "Networks." Click on "Spice client folder" once, and you will a 2nd "Spice client folder" entry appear. This new entry will have a folder icon, and has an eject icon on the far right (see the photo below). If you click on this new "Spice client folder" entry, or click the "Spice client folder" item listed on the left hand side and you will see your shared folder inside.
```

Comment:
Same as comment above. I realize this would be a lot to go back and change since the entire article is written this way, but just something to consider for future content.